src/backend/functions/change-requests-review.ts

```diff
@@ -23,28 +27,20 @@ export const reviewChangeRequest: Handler = async ({ body }, _context) => {
   if (!foundCR) return buildNotFoundResponse('change request', `#${crId}`);
 
   // update change request
-  const update = prisma.change_Request.update({
+  const update = await prisma.change_Request.update({
```

Comment:
Maybe a try catch would be good here to ensure proper error handling. We do not want to send message success on error.